### PR TITLE
Command to Delete Old Event Forums

### PR DESCRIPTION
**Pull request by cohoe**
This address #146 by introducing a management command that can be used in long-lived instances to clean up old event forums for events that are no longer in the schedule. The intention is that this will clean up some of the 500+ event forums in the beta (and my personal instance). This command will likely never be run on-boat or similar context.

### CODE REVIEW

File: Sources/swiftarr/Commands/PruneEventForums.swift [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+/// of forums starts to grow unweildy. The old events are marked as deleted and
+/// removed from the schedule, but the forums remain. This backend command is
+/// intended to be executed by an operator in one of these long-lived environments
+/// to soft-delete all forums that are linked to soft-deleted events.
+/// 
+/// The effects of this command can be reversed by importing the deleted schedule
+/// again which will restore the events and their associated forums (thanks, 
+/// soft-delete).
+///
+struct PruneEventForumsCommand: AsyncCommand {
```
Comment (hendricksond) [inline Sources/swiftarr/Commands/PruneEventForums.swift:18]: Nit: Indentation is weird in this file.

**Review by hendricksond — APPROVED** [review]
LGTM, and good idea too! Love data cleanup.

File: Sources/swiftarr/Commands/PruneEventForums.swift (hunk 1 above)
Comment (cohoe) [reply Sources/swiftarr/Commands/PruneEventForums.swift:18]: It is? I just ran it through `swift-format` (rules at https://github.com/jocosocial/swiftarr/blob/master/.swift-format) and it only adjusted one thing.

Comment (hendricksond) [reply Sources/swiftarr/Commands/PruneEventForums.swift:18]: It adjusted a bunch of indentation, see https://github.com/jocosocial/swiftarr/pull/312/commits/66715cebe9e65f1bd00c3249ba16ebb5765f2c68

But yeah, that took care of it!

**Pull request merged**